ReactiveCocoa/Swift/ObjectiveCBridging.swift

```diff
@@ -15,7 +15,7 @@ extension RACScheduler: DateSchedulerType {
 	}
 
 	public func schedule(action: () -> ()) -> Disposable? {
-		return self.schedule(action)
+		return (self.schedule(action) as RACDisposable) as Disposable?
```

Comment:
Totally, I'll separate each part.

I think, even if the method IS redundant, it serves as documentation. Otherwise users might wonder why this implements to `DateSchedulerType`.

Though now that I think about it, it probably is necessary since the return type is technically different.